packages/fast-components/src/card/card.styles.ts

```diff
@@ -0,0 +1,15 @@
+import { css } from "@microsoft/fast-element";
+
+export const CardStyles = css`
+    :host {
+        --elevation: 4;
+        --height: "100%";
+        --width: "100%";
+        display: block;
+        background: var(--neutral-layer-card);
+        border-radius: calc(var(--elevated-corner-radius));
+        width: var(--width);
+        height: var(--height);
```

Comment:
Just curious, are these values coming from the design system and you are reassigning the variables here? Why not just assign the height and width 100%?

Reply:
I agree - I'm not sure we really need to surface custom properties for this. An author can just as easily style the width / height properties directly

Reply:
I think there's 2 questions here with the underlying question of why is this here? :)

@eljefe223 these values aren't coming from the design system. They are custom properties I was exposing for the card. 

@nicholasrice I had a thought that we might be able to set the custom property from once from a parent and have all children respond accordingly, but that's not actually possible. If it were, I think it would've been a helpful use case. Since it doesn't do what I expected, it's terrible.
```html
    <div style="--width: 300px; --height: 163px"> // doesn't work, bad idea :)
        <fast-card>Foo</fast-card>
        <br>
        <fast-card>Bar</fast-card>
        <br>
        <fast-card>Baz</fast-card>
    </div>
```

Reply:
yea you're defining the property in the host. You might try:

```css
:host {
   height: var(--hard-height, 200px);
   width: var(--hard-width, 200px);
}
```

Reply:
That does provide a nice utility using the style attribute over creating a real css selector:

```css
.foo fast-card { height: x; width: y }
```

Reply:
I do like it - I'm open to exploring that if you are. I don't think it's a ton of overhead.

Reply:
Took a shot at using the custom variables here. Allows quite a bit of flexibility in terms of setting width/height. You can now use the vars in the parent, or you can define via class override, etc. Lots of utility to set width/height which I think if a goal for cards.

Reply:
Yea I'm open to it *I think* - my concern is using very generic terms like "height" and "width", and specifically I'm worried about un-intentional propagation of those properties to child elements that might use the same names.

For instance, both this card and the dialog PR implement the `--width` and `--height` custom properties. If I were to nest them and set the `--height` property on the dialog, the card would also implement that new value, which is likely undesriable.

Reply:
@nicholasrice should we prefix here with the component name? Seems safer and avoids the potential for the above conflict.

Reply:
It does - it also opens the door for the "why isn't *padding* parameterized, why isn't *insert property name here* parameterized" questions and were in the game of managing a bunch of component-specific custom property APIs.

While not 100% parallel, [this discussion](https://github.com/microsoft/fast-dna/issues/2685) does address managing component custom property APIs. It might be worth exploring this in a discussion format before setting the precedent in code. Where do we draw the line on what we want to expose as CSS custom properties?